Assets/Scripts/Actors/PlayerActor.cs

```diff
@@ -7,22 +7,34 @@ public class PlayerActor : Actor
 	ActorCamera _actorCamera = null;
 	PlayerActorStats _actorStats = null;
 	PlayerActorInventory _playerActorResources = null;
+	PlayerControls _controls = null;
+	Cutting _cutting = null;
 
 	public ActorCamera actorCamera
 	{
 		get { return _actorCamera; }
 	}
 
-	public PlayerActorInventory actorResources
+	public PlayerActorInventory inventory
 	{
 		get { return _playerActorResources; }
 	}
 
-	public PlayerActorStats actorStats
+	public PlayerActorStats stats
 	{
 		get { return _actorStats; }
 	}
 
+	public PlayerControls controls
+	{
+		get { return _controls; }
+	}
+
+	public Cutting cutting
```

Comment:
I actually changed these to rename remove the "player" prefix from their name. In the context they're being used you know that you're accessing the members of `PlayerActor`, so doing `PlayerActor.playerPhysics` is redundant compared to `PlayerActor.physics`. I figured this is inline with how Unity handles things like this, such as how `Random.RandomRange()` is deprecated in favor of `Random.Range()`.